.github/workflows/lint-images-base.yaml

```diff
@@ -23,10 +23,11 @@ jobs:
     name: Lint image build logic
     runs-on: ubuntu-24.04
     steps:
-      - name: Checkout default branch (trusted)
+      - name: Checkout base branch (trusted)
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
-          ref: ${{ github.event.repository.default_branch }}
+          # We first check if base_ref exist, meaning we're in pull_request_target context, and if not we just use ref_name as it's defaulted to main
```

Comment:
This workflow doesn't trigger via `pull_request_target`, so we should update the comment to be clear.